README.md

```diff
@@ -1 +1,131 @@
 Check: [Amending an versioning of dictionaries](https://lifecycle-project.github.io/ds-upload/articles/dictionaryVersioning.html)
+
+# Create a dataset from existing dictionary
+
+The transformation (or creating a dataset from existing dictionary) is performed once and is described here only to explain the process.
+
+The original dictionaries are located here: `lifecycle-project/ds-dictionaries/dictionaries/`.
+
+These dictionaries are saved in an Excel file with two sheets *Variables* and *Categories* and contain the following columns:
+
+*Variables*
+
+-   name
+
+-   valueType
+
+-   unit
+
+-   label
+
+*Categories*
+
+-   variable
+
+-   name
+
+-   isMissing
+
+-   label
+
+The format is roughly the format Opal uses for the dictionary template. Excel files are not version controlled in github. This is why new dictionaries will be saved as comma separated files (.csv) in order to enable version control.
+
+Please note that the original dictionaries (Excel) will not be saved as .csv files, instead these will be available by calling `data(lifecycle_core_3_0)` after loading the R package `library(dsDictionaries)`. Newly created dictionaries will be available as .csv for the developers/maintainers, users will be able to use the dictionary as described above. For example `data(athlete_urban_2_0)`.
+
+Data (dictionaries) are prepared under the folder `data-raw/`.
+
+The dictionary Excel files are saved in the folder `data-raw/{project}/{dictionary-kind}/{version}/`.
```

Comment:
Is this sustainable or does it get cluttered?

data-raw/{lifecycle}/{lifecycle_core_x_y}/{dictionary-kind}/{version}/
data-raw/{lifecycle}/{lifecycle_methyl_x_y}/{dictionary-kind}/{version}/

makes navigating probably easier.

Reply:
Good point, however I changed the structure to:

data_raw/{project}_{dictionary-kind}_{version}/